contracts/src/v0.8/ccip/tokenAdminRegistry/RegistryModuleOwnerCustom.sol

```diff
@@ -44,6 +47,20 @@ contract RegistryModuleOwnerCustom is ITypeAndVersion {
     _registerAdmin(token, IOwner(token).owner());
   }
 
+  /// @notice Registers the admin of the token using OZ's AccessControl DEFAULT_ADMIN_ROLE.
+  /// @param token The token to register the admin for.
+  /// @dev The caller must have the DEFAULT_ADMIN_ROLE as defined by the contract itself.
+  function registerAccessControlDefaultAdmin(
+    address token
+  ) external {
+    bytes32 defaultAdminRole = AccessControl(token).DEFAULT_ADMIN_ROLE();
```

Comment:
The oz access control system has default admin role hardcoded as a constant and isn't virtual, couldnt we save a bunch of gas by just re-using the constant here instead of making an external call?

Reply:
This is only used once and it shouldn't be an expensive call. It's not virtual but technically people could still change it. I don't see any harm in doing it this way